sklearn/ensemble/voting.py

```diff
@@ -279,7 +279,9 @@ def fit(self, X, y, sample_weight=None):
         self.le_ = LabelEncoder().fit(y)
         self.classes_ = self.le_.classes_
         transformed_y = self.le_.transform(y)
-
+        self.n_features_out_ = len(self.estimators)
```

Comment:
same comment as `forest.py`

Reply:
I think the only consistent thing is to have it after ``fit``, we can't know before. So you should comment on the ones that have it before ;)